tool/update_goldens.sh

```diff
@@ -4,9 +4,19 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-export DART_VM_OPTIONS="-DUPDATE_GOLDENS=true"
+echo "Checking flutter version..."
+if flutter --version | tee /dev/tty | grep -q 'channel stable'; then
+	export DART_VM_OPTIONS="-DUPDATE_GOLDENS=true -DGOLDENS_SUFFIX=_stable"
```

Comment:
Unless we have significant reason to diverge from published google style can we indent with 2 spaces here?

https://google.github.io/styleguide/shell.xml#Indentation

Reply:
Thanks for improving our shell style guide conformance! :)

Reply:
No good reason (except that I haven't read the style guides, and my VS Code defaults everything except Dart to tabs 🙃)

Fixed!